stories/point.stories.jsx

```diff
@@ -7,6 +7,7 @@ function DemInfoTestComponent(props) {
   const { vState } = props
   const theme = Theme
   return (
+    <div class="sk_bg big">
```

Comment:
`<div class="sk_bg big">` should be in the story.  You should create a story with vState: 'loading' to exercise this feature.